app/src/main/java/org/openmrs/eip/app/utils/ReceiverMessageUtils.java

```diff
@@ -0,0 +1,91 @@
+package org.openmrs.eip.app.utils;
+
+import java.text.MessageFormat;
+import java.time.LocalDateTime;
+
+import org.apache.camel.ProducerTemplate;
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.eip.app.management.entity.AbstractEntity;
+import org.openmrs.eip.app.management.entity.ReceiverSyncRequest;
+import org.openmrs.eip.app.management.entity.SenderSyncResponseModel;
+import org.openmrs.eip.app.management.entity.SyncMessage;
+import org.openmrs.eip.component.Constants;
+import org.openmrs.eip.component.SyncProfiles;
+import org.openmrs.eip.component.utils.JsonUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile(SyncProfiles.RECEIVER)
+public class ReceiverMessageUtils {
+	
+	private static final Logger log = LoggerFactory.getLogger(ReceiverMessageUtils.class);
+	
+	@Value("${" + Constants.PROP_CAMEL_OUTPUT_ENDPOINT + "}")
+	private String endpointConfig;
```

Comment:
Don't make this a property, just hard code like we do for the sender

Reply:
Yes I would like to give admins as much power to configure things but we also need to sort of limit it, the more configuration  options they have the more errors they are likely to make